packages/@react-stately/dnd/src/useDroppableCollectionState.ts

```diff
@@ -90,10 +130,12 @@ export function useDroppableCollectionState(props: DroppableCollectionStateOptio
 
       return false;
     },
-    getDropOperation(target, types, allowedOperations) {
-      return typeof props.getDropOperation === 'function'
-        ? props.getDropOperation(target, types, allowedOperations)
-        : allowedOperations[0];
+    // TODO: how do we feel about providing isInternalDrop and dragged keys to the user provided getDropOperation as well? That way allow the user to
+    // define if they'd like to allow dropping items on themselves
```

Comment:
Passing `draggingKeys` is a bit weird IMO. Definitely shouldn't be there if `isInternalDrop` is false, and I'm not sure about it for internal ones too. Seems like something a user could track themselves if needed. Unless it comes up as a common request, let's omit it for now.

Reply:
Fair enough, I'll go ahead and get rid of it